web/src/store/reducers/datasets.ts

```diff
@@ -32,7 +32,7 @@ export type IDatasetsState = {
   totalCount: number
   init: boolean
   deletedDatasetName: string
-  refreshTags: boolean
+  refreshTags: string
```

Comment:
Why would this be a string and not a boolean?

Reply:
So the thinking is twofold - one to bring it into line with

https://github.com/MarquezProject/marquez/blob/78a191b3cb73a50dc7466103e0588e80c3374042/web/src/components/datasets/DatasetDetailPage.tsx#L94

where we are refreshing based on the change in dataset name.

Second reason was for trouble shooting purposes.  Based on the concatanation of namespace/datasetname/field/tag it was just easier to understand/diagnose what was happening.

Reply:
Can always change the dependency to 
```
datasets.refreshTags === true
```
and change the reducer to set refreshTags to False when its updating then True on success.

Reply:
Ok, let's just go with what you have here.